prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -87,36 +91,44 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
             // run our call to the endpoint in a blocking fashion
             runBlocking {
                 launch {
+                    // parse headers for any dynamic values, OK needs the report ID
```

Comment:
I thought, launch(), is for async execution.  Why it says "runBlocking" which is sync execution?  Can we rename to runNonBlocking?  Perhaps, I miss something.